functions/nodejs-apps.bash

```diff
@@ -60,6 +60,20 @@ nodejs_setup() {
 frontail_setup() {
   local frontailBase
   local frontailUser="frontail"
+  local chosenTheme
+  local frontailTheme
+
+  # ask for light or dark theme
+if [[ -n $INTERACTIVE ]]; then
+    chosenTheme=$(whiptail --title "openHAB Log Viewer (frontail)" --menu "select theme" 18 116 2 --cancel-button Back --ok-button Execute \
+    "21 | light theme"           "frontail light theme, the openHABian default" \
+    "22 | dark theme"            "frontail dark theme" \
```

Comment:
I'd like you to do that in menu.bash to avoid the need for a user to choose twice.
Make the theme a parameter to frontail_setup()
Pay attention it keeps working even if called without a parameter (then use the default theme)

Note menu option 22 is in use already so do not use it.
See menu.bash how to have another, _unnumbered_ menu item right below 21